docs/css-style-guide.md

```diff
@@ -109,21 +115,17 @@ Where 'component' is a re-usable building block (eg. a tab bar),
 }
 ```
 
-## Colors and Typography
-
-- Use colors and typography values defined in a shared `variables` module
-  rather than hard-coding colors in component styles.
-
-## Layout
-
-- Use flexbox instead of floats for component layout. This avoids the need for clearfix and other hacks. Note that flexbox [may not be appropriate](https://hyp.is/AVKcwo8BvTW_3w8LypJ5/jakearchibald.com/2014/dont-use-flexbox-for-page-layout/) for overall page layout
-
-## Vendor Prefixes
-
-- Avoid vendor prefixes. Use [autoprefixer](https://github.com/postcss/autoprefixer) to add those.
+- Use color, typography and other values defined in the shared `variables` module
+  rather than hard-coding CSS rule values in component CSS
+- We use flexbox instead for layout; there are some utility mixins and classes
+  to make component styling more convenient.
+- Avoid vendor prefixes ([autoprefixer](https://github.com/postcss/autoprefixer) will
+  add these automatically).
 
 ## Browser Support
 
+TODO needs update
+
 - Hypothesis projects should be functional on Internet Explorer 11 and above. CSS features that are only available in newer browsers can still be used but the site should behave gracefully in older browsers
```

Comment:
Can you update this section based on https://web.hypothes.is/help/which-browsers-are-supported-by-hypothesis/? It says:

> The Hypothesis Chrome extension, Bookmarklet, and LMS application work within any modern web browser (a version of Chrome, Firefox, Safari, Microsoft Edge, or equivalent released within the last 12 months). Note that as of 1 July 2020, Hypothesis will no longer support Internet Explorer.